src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -162,8 +162,12 @@ function WorkspaceWorkflowsPage({policy, betas, route, reimbursementAccount, ses
                 subtitle: translate('workflowsPage.makeOrTrackPaymentsDescription'),
                 onToggle: (isEnabled: boolean) => {
                     let newReimbursementChoice;
+
                     if (!isEnabled) {
                         newReimbursementChoice = CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_NO;
+                    } else if (hasVBA && !Policy.isCurrencySupportedForDirectReimbursement(policy?.outputCurrency ?? '')) {
```

Comment:
I guess the only problem with this is that it will block proceeding with setting up indirect reimbursement if following the same steps from [my comment](https://github.com/Expensify/App/pull/38318#issuecomment-2007544821).

Reply:
If you have a VBBA set up, you shouldn't see indirect reimbursement step with `Connect a bank account` from my understanding.

Reply:
I meant more that you should still be able to toggle `Make or track payments` on (which enables indirect reimbursement), whether or not your currency is supported. 

I actually can't follow [my steps](https://github.com/Expensify/App/pull/38318#issuecomment-2007544821) anymore (disabling reimbursements seems to not work on OD even), so we don't necessarily need to block on this. Maybe we can handle it in a follow-up, since it's a pretty specific edge case since you have to originally have a workspace with USD + VBBA set up.

Reply:
Understood, I've updated the logic does it look good to you?

By the way, under the hood when we call the API with `CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_NO`, it sets it to manual/indirect.